README.md

```diff
@@ -95,12 +95,23 @@ the instructions in [podman.md](podman.md).
 To run the tests, type `./gradlew test` from the `hibernate-reactive` 
 directory.
 
+#### Starting databases automatically with Testcontainers
+
+If you have Docker installed, the container instances will be automatically started
+by running the tests if you select the `-Pdocker` profile like so:
+
+    ./gradlew test -Pdocker
+    
+TIP: To reuse the same containers across multiple runs, set `testcontainers.reuse.enable=true` in
+a file located at `$HOME/.testcontainers.properties` (create the file if it does not exist).
+
```

Comment:
In the future could you keep these changes in a separate commit? These doesn't seem related to DB2. It's more an improvement of the documentation.